pkg/ovssubnet/controller/kube/bin/openshift-ovs-subnet

```diff
@@ -34,13 +34,16 @@ Setup() {
     source /etc/openshift-sdn/config.env
     cluster_subnet=${OPENSHIFT_CLUSTER_SUBNET}
 
-    pid=$(docker inspect --format "{{.State.Pid}}" ${net_container})
     network_mode=$(docker inspect --format "{{.HostConfig.NetworkMode}}" ${net_container})
     if [ "${network_mode}" == "host" ]; then
       # quit, nothing for the SDN here
       exit 0
+    elif [[ "${network_mode}" =~ container:.* ]]; then
+      # Get pod infra container
+      net_container=$(echo ${network_mode} | cut -d ":" -f 2)
```

Comment:
why is this needed? shouldn't all containers in the pod show the same IP address?